static/search.js

```diff
@@ -506,25 +506,32 @@ var Search = Control.extend({
 			numResults: numResults,
 			searchValue: this.searchTerm
 		},{
-			docUrl: function(){
+			docUrl: function(url){
 				if(self.options.pathPrefix){
-					return self.options.pathPrefix + "/" + this.url;
+					return self.options.pathPrefix + "/" + url;
 				}
 
-				if(this.url.substr(-1) === "/"){
+				if(url.substr(-1) === "/"){
 					return this.url;
 				}
 
-				return "/" + this.url;
+				return "/" + url;
 			},
-			addTargetToExternalURLs: function(html){
+			addTargetToExternalURLs: function(html, parentHref){
 				var $html = $('<div>').html(html);
 				$html.find('a').each(function(){
 					var $a = $(this);
 					var a = $a[0];
```

Comment:
Going through this dance to convert the object to be jQuery-wrapped and then to unwrap it seems odd.